geospace-lib/src/main/scala/com/socrata/geospace/lib/regioncache/SpatialIndex.scala

```diff
@@ -1,7 +1,9 @@
-package com.socrata.geospace.regioncache
+package com.socrata.geospace.lib.regioncache
 
-import com.socrata.geospace.feature.FeatureExtensions._
-import com.socrata.geospace.Utils._
+import com.socrata.geospace.lib.Utils
+import com.socrata.geospace.lib.feature.FeatureExtensions
+import FeatureExtensions._
+import Utils._
```

Comment:
By the way, it's better to not do relative imports like `FeatureExtensions._`, but include the full path instead for clarity.